packages/inference/src/tasks/multimodal/documentQuestionAnswering.ts

```diff
@@ -0,0 +1,49 @@
+import { InferenceOutputError } from "../../lib/InferenceOutputError";
+import type { BaseArgs, Options } from "../../types";
+import { request } from "../custom/request";
+
+export type DocumentQuestionAnsweringArgs = BaseArgs & {
+	inputs: {
+		/** Base64 of document image **/
+		image: string;
```

Comment:
```suggestion
		/** 
		 * Raw image
		 * 
		 * You can use native `File` in browsers, or `new Blob([buffer])` in node, or for a base64 image `new Blob([btoa(base64String)])`, or even `await (await fetch('...)).blob()`
		 **/
		image: Blob;
```

I would rather use native `Blob` as the datatype for the image, even if we later convert it to base64 to pass to the inference API.

Reply:
So we'll do blob to base64 conversion for user?
Perhaps we should still allow base64 string, to avoid base64 decoding in case they already have base64 for some reason

Reply:
> Perhaps we should still allow base64 string, to avoid base64 decoding in case they already have base64 for some reason

Not a fan, it's an optimization for a specific use case. We can revisit later but for now I'd rather just the `Blob` like our other tasks, to keep the API straightforward

Reply:
by the way there's already a `base64FromBytes` function that can be moved to `packages/shared`